subsys/bluetooth/controller/ll_sw/ull_llcp_conn_upd.c

```diff
@@ -196,6 +196,22 @@ static bool cu_check_conn_parameters(struct ll_conn *conn, struct proc_ctx *ctx)
 }
 #endif /* CONFIG_BT_CTLR_CONN_PARAM_REQ */
 
+static bool cu_check_conn_ind_parameters(struct ll_conn *conn, struct proc_ctx *ctx)
```

Comment:
Suggest to rename `check` to `valid` or something. 

If a `check` function returns true, is it really obvious that it's valid?

IMO `check` works better with `int` than `bool`